docs/UserGuide/QuickStart/ClusterQuickStart.md

```diff
@@ -0,0 +1,173 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+    
+        http://www.apache.org/licenses/LICENSE-2.0
+    
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+## Quick Start
+
+This article uses a local environment as an example to
+illustrate how to start, expand, and shrink a IoTDB Cluster.
+
+### 1. Prepare the Start Environment
+
+Unzip the apache-iotdb-1.0.0-all-bin.zip file to cluster0 folder.
+
+### 2. Start a Minimum Cluster
+
+Start the Cluster version with one ConfigNode and one DataNode(1C1D), and
+the default number of replicas is one.
+```
+./cluster0/sbin/start-confignode.sh
```

Comment:
```suggestion
bash ./cluster0/sbin/start-confignode.sh
```
Maybe adding bash is better, because some Linux system may not allow direct execution